### DISCUSSION TITLE
correct node.js version to install

### DISCUSSION THREAD

**Discussion by rodolani** [Lissy93/dashy#1480] [Help and Support]
Hi, I installed this app following the instructions in the deploy.md section and downloaded it with git. But in the "Build from source" section: "latest or LTS version of Node.js installed" is specified while in the app.json file it is
   "engines": {
     "node": ">=16.0.0 <18.0.0"
What version of nodeJS?
The software has some problems, which perhaps I can then specify but if I use the right version of nodejs perhaps I think things can improve.

Thanks in advance for the reply.

**Comment by lissy93** [comment]
This will be addressed in the next PR:

https://github.com/Lissy93/dashy/blob/dab344044819b702a15003c41452b0d4cd2176ab/package.json#L68-L70

Branch: [`FEAT/2.1.2_dependency-updates-n-fixes`](https://github.com/Lissy93/dashy/blob/FEAT/2.1.2_dependency-updates-n-fixes)

**Reply by lissy93** [reply]
And the version the Dockerfile will use, will be the current LTS (which I think is 20.11.1)
